tests/conftest.py

```diff
@@ -21,6 +23,20 @@ def begin(conn):
         conn.execute('BEGIN')
 
 
+@pytest.fixture
+def event_loop(request):
```

Comment:
Having `-e .[test]` in `dev-requirements.txt` will install sqlalchemy_aio and all of its dependencies